plumbing/chn/chain_test.go

```diff
@@ -46,6 +47,10 @@ func (mcr *FakeChainer) Head() types.TipSet {
 	return mcr.head
 }
 
+func (mcr *FakeChainer) LatestState(ctx context.Context) (state.Tree, error) {
+	return nil, errors.New("Not defined yet")
```

Comment:
All plumbing code should be tested.

Reply:
`ChainGetActorFromLatestState` is just a pass through method to the chain. There's not really anything to test here except "method calls other method," and chain already has it's own tests.